### PR TITLE
Move discord automation into ci.yml

### PR DESCRIPTION
**Pull request by jgomez720**

### CODE REVIEW

**Comment by pierremtb** [comment]
Oh dope!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -370,3 +370,27 @@ jobs:
...
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install requests
+  
+      - name: Announce Release
+        env:
+          DISCORD_WEBHOOK_URL: ${{ secrets.DISCORD_WEBHOOK_URL }}
+          RELEASE_VERSION: ${{ github.event.release.tag_name }}
+          RELEASE_BODY: ${{ github.event.release.body}}
```
Comment (pierremtb) [inline .github/workflows/ci.yml:395]:
```suggestion
          RELEASE_BODY: ${{ github.event.release.body }}
```

**Review by pierremtb — APPROVED** [review]
Let's go! Feel free to ignore the tidbit

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -370,3 +370,27 @@ jobs:
         uses: softprops/action-gh-release@v1
         with:
           files: 'artifact/*/Zoo*'
+          
+  announce_release:
+    needs: [publish-apps-release]
```
Comment (Irev-Dev) [inline .github/workflows/ci.yml:375]:
```suggestion
    needs: [publish-apps-release]
    if: ${{ github.event_name == 'release' || github.event_name == 'schedule' }}
```

@pierremtb might it be worth adding the same if statement that `publish-apps-release` has, or is teh `needs` enough to cover that?

**Review by Irev-Dev — APPROVED** [review]
This looks perfect.

I guess we won't know until we merge and do the next release, but that's fine.

I did have one question for Pierre, but I doubt its needed.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (pierremtb) [reply .github/workflows/ci.yml:375]: needs should do the trick (if you check the last commit, you'll see that both are shown as skipped)

**Pull request merged**